internal/service/elasticache/replication_group_test.go

```diff
@@ -66,6 +66,102 @@ func TestAccElastiCacheReplicationGroup_basic(t *testing.T) {
 	})
 }
 
+func TestAccElastiCacheReplicationGroup_network_type_ipv6(t *testing.T) {
```

Comment:
You can rename the function to `TestAccElastiCacheReplicationGroup_networkTypeIPV6`

Reply:
Completed